docs/guide/monitoring-guide.md

```diff
@@ -217,7 +217,7 @@ Just like with the Events toggle, turning on the **Logs toggle** in the Reports
 
 Clicking on the **Logs icon** will open **Logs Table component** below the Logs Search component. A list view of log entries will be displayed with `@timestamp`, `_source`, and `Tags` as default columns, but all available log fields will be available in the **Logs Table component dropdown menu**, and easily selected or deselected and displayed in data table view. Column width can be also resized to better organize Log Table when multiple fields are selected at once.
 
-Every Log entry in the table has two left aligned icons, one that opens **panel with all log details**, and the other that opens **Log context**. Contextual logging is an approach that encourages not just adding additional useful data to log events, but also sharing that data across related events. Check out the [Logs Guide](./logs-guide) for more info. 
+Every Log entry in the table has two left aligned icons, one that opens **panel with all log details**, and the other that opens **Log context**. Contextual logging is an approach that encourages not just adding additional useful data to log events, but also sharing that data across related events. Check out the [Logs Guide](../logs) for more info. 
```

Comment:
Log entry => log event
left aligned => left-aligned

Re "Contextual logging is...."
I think that is just not correct.  "Log Context" is simply a display of N log events before and after a given log event, typically used to help with troubleshooting.  The Log Context functionality "ignores" filters and shows surrounding log events even if they don't match the currently used filters/query.

Reply:
Added this text to the other paragraph in the other file. It's not needed here.